src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -9,7 +9,7 @@ import AdSlot from './AdSlot';
 const LEADERBOARD_HEIGHT = '5.5rem';
 const LEADERBOARD_HEIGHT_GROUP_4_5 = '9rem';
 
-const StyledAdDiv = styled.div`
+const StyledAdDiv = styled.div.attrs({ ariaHidden: 'true' })`
```

Comment:
Please change to section element with region role and labe as per comment https://github.com/bbc/simorgh/pull/7025#discussion_r450155348